sdk/canton/BUILD.bazel

```diff
@@ -1425,6 +1431,8 @@ proto_jars(
     maven_artifact_prefix = "ledger-api",
     maven_artifact_scala_suffix = "scalapb",
     maven_group = "com.daml",
+    proto_deps = ["//daml-lf/ledger-api-value:ledger_api_value_proto"],
+    scala_deps = ["//daml-lf/ledger-api-value:ledger_api_value_proto_java"],
```

Comment:
why is this a Scala dep?

Reply:
I think it is because scala classes need access to all the java pbs. Please note that I had to add support for `scala_deps` as part of this PR. This attribute was defined but was being ignored by the `proto_jars` routine

Reply:
> Please note that I had to add support for scala_deps as part of this PR
I noticed that. What looks strange is that you add something that has `_java` in the name as a Scala dependency. Mostly in other places PB sources ending with `_java` are added just to `deps`